src/coprocessor/endpoint.rs

```diff
@@ -426,8 +427,11 @@ impl<E: Engine> Endpoint<E> {
         // There might be errors when handling requests. In this case, we still need its
         // execution metrics.
         let mut storage_stats = Statistics::default();
+        let mut exec_summary = ExecSummary::default();
+        handler.collect_scan_stats(&mut exec_summary);
```

Comment:
You may collect nothing here since https://github.com/tikv/tikv/blob/44b5446549044a1ce11eb9053fe14971abc28e07/components/tidb_query_executors/src/runner.rs#L464 
removes all stats.

Reply:
Extract from `result` instread?

Reply:
You mean the exec_stats is empty in `handle_unary_request_impl` ? 
I need to get the exec_stats from `rel_resp` in `handle_request` in runner.rs?

Reply:
How about put the `self.exec_stats.clear();` to `collect_kv_read_time()`? : )